clients/da-vinci-client/src/main/java/com/linkedin/davinci/client/AvroGenericDaVinciClient.java

```diff
@@ -766,9 +768,7 @@ public synchronized void start() {
         daVinciConfig::getRecordTransformer);
 
     try {
-      if (!getBackend().compareCacheConfig(cacheConfig)) {
-        throw new VeniceClientException("Cache config conflicts with existing backend, storeName=" + getStoreName());
```

Comment:
Can we clean up `compareCacheConfig` method?

Reply:
Oh yeah for sure. Done in the latest commit. Thanks!